stdlib/Printf/src/Printf.jl

```diff
@@ -73,115 +73,116 @@ end
 base(T) = T <: HexBases ? 16 : T <: Val{'o'} ? 8 : 10
 char(::Type{Val{c}}) where {c} = c
 
-# parse format string
-function Format(f::AbstractString)
-    isempty(f) && throw(ArgumentError("empty format string"))
-    bytes = codeunits(f)
-    len = length(bytes)
-    pos = 1
-    b = 0x00
+@inline function _detectFormat!(f, bytes, len, pos::Int64, fmts::Vector{Spec})
```

Comment:
```suggestion
@inline function _detectFormat!(f, bytes, len, pos::Int, fmts::Vector{Spec})
```
That's probably why it fails on 32 bit